Chummer/Backend/Characters/Character.cs

```diff
@@ -2697,7 +2697,7 @@ public bool Save(string strFileName = "", bool addToMRU = true, bool callOnSaveC
 
             if (callOnSaveCallBack)
             {
-                blnErrorFree = DoOnSaveCompleted.Aggregate(blnErrorFree, (current, funcToRun) => funcToRun(this) && current);
+                blnErrorFree = DoOnSaveCompleted.ToArray().Aggregate(blnErrorFree, (current, funcToRun) => funcToRun(this) && current);
```

Comment:
In theory, this will fix the issue of the crash. However, it will mean that any `DoOnSaveCompleted` functions that add extra functions to the end of `DoOnSaveCompleted` wouldn't work. I don't know how important that would be though.

Reply:
I think we can live with that.